doc/methods/reprocessing.rst

```diff
@@ -3,4 +3,162 @@
 Reprocessing Methods
 ====================
 
-Coming soon!
+SaltProc models separations and feeds using three different data structures:
+
+Material flows
+--------------
+A material flow represents a material with a given
+volume, density, temperature, and nuclide composition.
+It can also include information such as void fraction. :class:`saltproc.Materialflow`
+objects contain the prior mentioned quantities as attributes, as well as
+methods to perform the following:
+
+1. Add two :class:`saltproc.Materialflow` objects together
+
+   a. The mass of the sum is :math:`m_{1+2} = m_{1} + m_{2}`
+
+   b. The composition of the sum is :math:`N_{1+2} = \frac{N_{1} * m_{1} + N_{2} * m_{2}}{m_{1} + m_{2}}`
```

Comment:
Consider including in the above paragraph if `N` and `m` are vectors, scalars, etc.